x-pack/plugins/alerting/common/alert_summary.ts

```diff
@@ -32,6 +32,7 @@ export interface AlertSummary {
 }
 
 export interface AlertStatus {
+  uuid?: string;
```

Comment:
In which case will `uuid` be optional?

Reply:
This is kind of a funky data structure.  It sometimes contains a "muted" indicator for alerts which are not currently active, and so don't also have an `actionGroupId`, and won't have a `uuid`.  Beyond that, existing rule's task state when migrated to this Kibana version will not have the `uuid` field set, so I wanted to account for that, typing-wise.